### ISSUE TITLE
🐛 Bug Report: Default Value not Saved When Creating String/Integer/Float/Boolean Attribute with falsy default value

### ISSUE THREAD

**Issue by McPizza0** [appwrite/appwrite#4712]
### 👟 Reproduction steps

Use Console 2.0
Go To Collection
Create new Attribute
Select type of INT or FLOAT
Set Default value to a 0 (or 0.0 for float)
Save Attribute

### 👍 Expected behavior

Attribute should be created with the default value saved
On document creation, default value should be applied

### 👎 Actual Behavior

Default value is not saved on attribute creation
New documents do not use the default value

### 🎲 Appwrite version

Version 1.0.x

### 💻 Operating system

MacOS

### 🧱 Your Environment

Local Docker

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by greendesertsnow** [comment]
also the workaround doesn't work
```js
sdk.Query.notEqual(attribute, true),
```
will not bring falsy values.

**Comment by stnguyen90** [comment]
I tried to create a boolean attribute with default value of false, but this was the payload:

```json
{
    "key": "b",
    "required": false,
    "array": false
}
```

**Comment by stnguyen90** [comment]
And strings seem funky too. Even if i fill in text and then delete all the text to try and have an empty string be the default value, the payload is:

```json
{
    "key": "empty_default",
    "size": 25,
    "required": false,
    "array": false
}
```

**Comment by abnegate** [comment]
Fixed in 1.3.0
